.github/workflows/build_test.yml

```diff
@@ -3,45 +3,30 @@ name: ROS2 CI
 on: [pull_request, push]
 
 jobs:
-  test_environment:
-    runs-on: [ubuntu-latest]
+  build-and-test:
+    runs-on: ubuntu-latest
     strategy:
       fail-fast: false
       matrix:
         ros_distribution:
-          - foxy
           - galactic
-          - rolling
```

Comment:
Could you reenable rolling? Thanks.

Reply:
Addressed [here](https://github.com/ros-drivers/transport_drivers/pull/84/commits/32943067c12da460a98d4792e5cad3914ae8d2b7)